html/browsers/origin/cross-origin-objects/cross-origin-objects.html

```diff
@@ -187,7 +187,7 @@
   var isSymbol = (typeof(propName) == "symbol");
   propName = String(propName);
   assert_true(isObject(desc), "property descriptor for " + propName + " should exist");
-  assert_equals(desc.enumerable, false, "property descriptor for " + propName + " should be non-enumerable");
+  assert_equals(desc.enumerable, true, "property descriptor for " + propName + " should be enumerable");
```

Comment:
This change is wrong.  It should be comparing desc.enumerable to !isSymbol, because per spec the symbols should still be non-enumerable.